From dea1bd2917ed9490ee96162288e0904e3de9da2d Mon Sep 17 00:00:00 2001
From: Romain Naour <romain.naour@gmail.com>
Date: Tue, 16 Feb 2016 00:24:30 +0100
Subject: [PATCH] build-sys: fix typo

When timer_create is available have_timer must be
set to "yes". But instead have_time is used.

Replace have_time by have_timer.

Fixes:
http://autobuild.buildroot.net/results/993/9935cd0522d4f978ba2e788a690f66790686b76b

Signed-off-by: Romain Naour <romain.naour@gmail.com>
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 4559573..3950f0c 100644
--- a/configure.ac
+++ b/configure.ac
@@ -390,7 +390,7 @@ AC_CHECK_FUNCS([clock_gettime], [],
 
 have_timer="no"
 AC_CHECK_FUNCS([timer_create],
-	[have_time="yes"],
+	[have_timer="yes"],
 	[AC_CHECK_LIB([rt], [timer_create], [
 		have_timer="yes"
 		REALTIME_LIBS="-lrt"
-- 
2.5.0

